cloud/issue.go

```diff
@@ -981,9 +981,49 @@ func (s *IssueService) AddComment(ctx context.Context, issueID string, comment *
 	return responseComment, resp, nil
 }
 
+// GetComment returns a comment for issueID and commentID.
+//
+// Jira API docs: https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-comments/#api-rest-api-2-issue-issueidorkey-comment-id-get
+func (s *IssueService) GetComment(ctx context.Context, issueID, commentID string) (*Comment, *Response, error) {
```

Comment:
Should we also support the optional parameter `expand`?

Reply:
[8d9dac0](https://github.com/andygrunwald/go-jira/pull/737/commits/8d9dac0f66d72ce09ba87ec0bb41cd381b3028e6)